tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -6442,6 +6446,8 @@ node_modules/chrome-devtools-frontend/front_end/heap_snapshot_worker/HeapSnapsho
 node_modules/chrome-devtools-frontend/front_end/heap_snapshot_worker/HeapSnapshot.js(918,34): error TS2345: Argument of type 'Uint32Array' is not assignable to parameter of type 'number[]'.
   Property 'pop' is missing in type 'Uint32Array'.
 node_modules/chrome-devtools-frontend/front_end/heap_snapshot_worker/HeapSnapshot.js(920,34): error TS2345: Argument of type 'Uint32Array' is not assignable to parameter of type 'number[]'.
+node_modules/chrome-devtools-frontend/front_end/heap_snapshot_worker/HeapSnapshot.js(1020,16): error TS2587: JSDoc type 'JSHeapSnapshotEdge' circularly references itself.
```

Comment:
I think this is a legitimate error, although it is hard to fllow; HeapSnapshot.createEdge references JSHeapSnapshotEdge, not HeapSnapshotEdge. Then JSHeapSnapshotEdge references JSHeapSnapshot, which circularly references HeapSnapshot.

I think the correct thing is for HeapSnapshot.createEdge to reference HeapSnapshotEdge. Or not use classes. That would work too.